test/python/test_apps.py

```diff
@@ -123,5 +275,122 @@ def test_hamiltonian(self):
         self.log.info(Energy_Estimate(data, pauli_list))
 
 
+class TestAppsFermion(QiskitTestCase):
+    """Tests for apps"""
+
+    def setUp(self):
+        self.j1 = 20
+        self.j2 = 3
+        self.j3 = 5
+        self.n = 10
```

Comment:
the js have to be < n

Reply:
Optional : raise an error in _set functions in fermion.py if this is violated. The condition is valid for all the three  _set functions

Reply:
I changed `j1` to `8`. Because that makes the call `flip_set(self.j1, self.n)` to return `[]`, I'm also adding the case with `j1 = 20`. See https://github.com/1ucian0/qiskit-sdk-py/commit/d5f2dfabaeeacd42a74226bb376dc0313d2bff0e